proto/mixings.proto

```diff
@@ -3,7 +3,19 @@ syntax = "proto3";
 package bayesmix;
 
 message DPState {
-    double totalmass = 1;
+    message FixedValue {
```

Comment:
I'm not sure this is a State anymore, rather DPParams? DPPrior?

For example in the Hierarchyies, the UnivariateLSState just collects mean and std, here instead you're also adding the prior information.

I would have two separate protos for consistency: DPState (as before) and DPParams/DPPrior (this one)
The DPMixing class will receive a DPPrior message in the constructor and write the current value of the totalmass to a DPState

Reply:
Why is DPState needed? The hierarchy classes contain private members mean and sd, and that's were the information is stored. The *LSState protos are only used for transportation purposes, specifically in the hierarchies' write_state_to_proto() members, which the mixtures lack. A DPParams class should be enough

Reply:
Never mind, I found it in marginal_state.proto. I'll be creating both DPState and DPParams then, even though the former is currently unused

Reply:
well we should use it in the collection phase of the chain, monitoring the totalmass parameter is usually extremely useful